terraform/standalone.tf

```diff
@@ -45,7 +45,7 @@ resource "aws_key_pair" "all" {
     // TODO: Same keys are also define in the FW Core Infrastructure State. Due to circular dependencies, and TF version conflicts I could not import those keys into this state
     //  we only need the keys here to add them to the bastion host. An alternative would be to create a separate bastion host for 3SC in their repo
     tyeadon_3sc  = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDEZ2a1o2OQCvScQipFvnQ7OrCxWRx7QwGa76BB6YJ9aex13AANeMXQQ3hLWdKvTA03N47x6CwbwBcFs532Oc0EFjYrFYmt3/ZrUW87OKC0LJz+i9Ap7HfMtJWAKL5HyFWTqL1ohsXrXftdotq54rfJK2xJ+hRsFVKXxd8FFVhPNAN5nV7oVf+7Q9/WnPwXcHJvPQCys6oiDCySk0a9P76sW1vSFghAIokgMsFYK9PE5gLP4wT3G13A+Z+VOZTLzUJHoYRnFK/QPI2P5fAf7vstVYwIdDhw9NwZF2j9bTabQsqJrxVUrqCX2A2xEzLgfbVQm4JG5LWxneLTkzX1vzHr"
-    oevans_3sc  = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCd3YO71r9bA5ziwi0upz1ESlDUKqgLeWRdROJ5hRNb7fkMx68UnHqPxj/S/+OXWMjlW1kSnSqXZcaWbqSkQ7neI6obMjaQ7lGxCy1NPPDzwv/BID4S2U3hMIMKoAlhK6P0rvSPkn4wpPl4g8Dlmj9y0nX2GBK3zcoeTroDA9EUtZspjTX/+3lcJS/Yln+ZVHtTQVT83HbFXWyui53TyRG2m1ieEcCCUFYxeSKFdQvSTqTD+AioXdU7Z/Akie4DR/J1o1rO3WlBvpYqSAnWOcj+l1VtJYE7xMr/O+L6CkfhuIoU/LlbagdEJsq03WAYUfETUCCTcwKn2ALHQ4bQ/TeCYuEfnZ2KpUZOY+goNpptXozKx1+SDjJjpXbZ4mZcawEmPYQQS/dcgQi40X038c/X7nxtnQNWJUbbwIhiZ+mdfiRy7CS6J1u7LRm5T17Vg+V5IlKW98tDmbx9TFzUXeODgDoqII9KoF79+E/WvHNuQNqIAC/DMIFoGaOMS1R30dM="
+    oevans_3sc  = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDdy9oatA/nDCdGpBusaK/XiqSWrWSMtVkBQAfIEznZf1vCUsbOrgLQTYJWbuPaHj0IBzZlEeFATLEY04GIYa++t1du1JAIJqd2bhNYrEGYPimkzF11k93UygIuYnDdgJfApwyibdvUj63xtgP/INzUJan2NdgGZ/pg7ZJAbPMZgtE+QO8qFZkHIsnnAyJl+ZyV5SrMjK+5Qxv05TuR+bb1sGg05IW2uqAHdRMZEREfRdDoo1jVU7oIsHbNJQdSvA6kC0NBfIn0M1nb/br6t+Gr7oACzpOs/JKSOinIi0l1pJZ1dDoQTS5ACppUh5MXjgsmGCxYk7pN7x0vTj+bxQBFLbJZklK/dTAPVO8MKIFDHgfnh4LyoEPnOcpkUcmZ3Dxl1PulEYmtQRkOdQPI5jkF2SOT/iJ42UIgMZ1m08ZOT4wf+oKncW9Rb/4uo+PHRddrdBcyS/dZHkLDogvYNpMXpeJyniRHcEsXszv5HPPf88Ka93pj0N1btz0bwgpG8Yc= owen@3s-MacBook-Pro.local"
     gcrosby_3sc  = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDlNQHV5VApZuneWtc9m9d7WEUqmfoLWm0John5vRwoPC0GYIU56BH90Yeiw5HkXJsiqnO+WXubFqWylhCRyfckNiTC7sKbpydZHVH4VmvNzOV4z8BXPob1qsnL2d+5eO8U7Sf21jpBQ4HEXgBk4GZ4eRuktM4eYRGsgTRW/FLFUex6c76Nb5va0FakDKXNKiojIoTIjLN0sxKAQtxuJAt4X4Jg6rtd5pS/4l9pH/VPncKcag1tDvx5ytN/4+lb9IZg/8OyG5JZDWaCsvhauJxn+LGP3GtHiEmiu3IMvTwthVWBj1rmFaX/KoOSlQazHlzEREHQ51mb+6MXSwoz+WrqcgkvFLtky0syMRqwjBgCU2IoKS/Cn2+qh7pI0L7ctPb7WjKmQw7vTfQDW3IDPPU2/H2WlJRChrLMWYzFt6oBWKDr4D7YwH89LYsA67rR9xZHY6TgmVexjiXPjnawAqHKEryESqSuNLDWQmNwrGJaWzmf04T3N+5puDIyuhq5MIlbP63mxSXOUEsFIsCKZPkuh/oR105cbSW3U2fZIajuNICXU/YETChn9K7CaR53uqWM7A6vU2VipNb8NJ4v0IP1djECR3/HwrCY+04Fvt/ZOzbvME6cXxfPZLCDRF9Styz4NiTKPQz/6g3Gbl6CF86vdG8uVKmLRUbSBUbEJX02Sw=="
   }
```

Comment:
can you please add my id_rsa.pub to the list. 
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDI3Vd+ksZNDpkxKLdwgHD5FZ8ngUk7Xvj9cV9nlGQ90ti77d+QtqRjGYpCmechizl5BEaaT9xi1DH8W7h3Isu85CnuAQqwb5lZMKDGsEzQTzxZ7h3AuMFkMNrN3d7PupwFifLVXmQF5R7E9I0EIGTrtnrINrzWMuU4VxVu3N0z3VDdkAvOAPDARaggr9K5zvmxB8NQ0iXCafNFk5bsddour/yxmWXxT7M3+qDB3CcHVzFqtMKVPyAs9HSuEfNBpenSTyNCMw78Bn7uvTzZdVlLIfmvz4H17pwKnoQTf3TEzTmpKg8A3XaqbHVrFODr7zl11tVxykA5nsy+FdeRu2z8quUji/qK0tStAd/1F6a19bLZ1rvlZp5uGbQnMuqNqiuEJs0F90VqcaEZ1wZe7d6EFPr++Yby2hUEo/eh3X8aSg50uog5S4f3pnbzB6RTB1dwLZWMo/tTx0UW/SavrbTcQlCOm31uKfJWkOgVW/lVikPq0O7k903OMIPPMSkCJ3U= iuripopov@Iuris-MacBook-Pro.local